tools/PI/DevHome.PI/Views/BarWindowVertical.xaml

```diff
@@ -80,11 +80,10 @@
 
             <controls:PISeparator Orientation="Horizontal"/>
 
-            <controls:ProcessSelectionButton x:Uid="ProcessChooserButton" x:Name="ProcessChooserButton" Command="{x:Bind _viewModel.ProcessChooserCommand}" HorizontalAlignment="Center" Margin="0">
-                <TextBlock Text="&#xecaa;" Margin="0" />
+            <controls:ProcessSelectionButton x:Uid="ProcessChooserButton" x:Name="ProcessChooserButton" Command="{x:Bind _viewModel.ProcessChooserCommand}" HorizontalAlignment="Center" Visibility="{x:Bind _viewModel.IsProcessChooserVisible, Mode=OneWay}">
+                <TextBlock Text="&#xe179;" />
```

Comment:
Doesn't need fontfamily/fontsize?

Reply:
Nope. In the StackPanel.Resources up above, there is a default style set for the textblocks. 

<Style TargetType="TextBlock">
    <Setter Property="FontFamily" Value="{StaticResource SymbolThemeFontFamily}"/>
    <Setter Property="FontSize" Value="{StaticResource SubtitleTextBlockFontSize}"/>
</Style>

I removed the unneeded declarations for the other buttons though. Thanks!